src/node_jsvmapi.h

```diff
@@ -116,80 +116,99 @@ enum napi_valuetype {
   napi_function,
 };
 
-// Environment
-NODE_EXTERN napi_env napi_get_current_env();
+enum napi_errorvalue {
```

Comment:
In the design discussion we had talked about a flag of some sort in the return code that would additionally indicate whether an exception is pending. I can't remember what the consensus was on that.